editor-core/src/main/webapp/app/core/editorCore/model/DefaultDiagramNode.ts

```diff
@@ -171,23 +203,27 @@ export class DefaultDiagramNode implements DiagramNode {
         return String(this.boundingBox.width) + ", " + String(this.boundingBox.height);
     }
 
+    changeTextPosition() : void {
+        var dx =  this.getX() - this.lastPointermoveCursor.x;
+        var dy =  this.getY() - this.lastPointermoveCursor.y;
+        console.log("Diagram pos in changeTextPosition : " + this.getX() + ", " + this.getY());
```

Comment:
We do not use log levels, so it's better to remove unnecessary logs before merge to master.